src/cockpit/channel.py

```diff
@@ -194,7 +201,7 @@ def do_pong(self, message):
     def do_open(self, options):
         self.receive_queue = asyncio.Queue()
         self.flow_control = options.get('flow-control') is True
-        asyncio.create_task(self.run(options), name='f{self.__class__.__name__}.run({options})')
+        asyncio.create_task(self.run(options), name='f{self.__class__.__name__}.run_wrapper({options})')
```

Comment:
Also `self.run_wrapper(options)` in the first argument?

Reply:
oops :)

Reply:
And maybe just `run` in the name?